include/lib/utils_def.h

```diff
@@ -160,5 +160,11 @@
  */
 #define ASSERT_SYM_PTR_ALIGN(sym) assert(((size_t)(sym) % __alignof__(*(sym))) == 0)
 
+/* Compiler builtin of GCC >= 9 and planned in llvm */
+#ifdef __HAVE_SPECULATION_SAFE_VALUE
```

Comment:
I'm told this flag is automatically defined by compiler versions that include the builtin, in which case the mitigations here will be on by default if such a compiler is used. This should be clearly stated in the commit message.

Reply:
Done.